src/Service/KeeperStore.h

```diff
@@ -95,7 +95,7 @@ class ConcurrentMap
         bool emplace(std::string const & key, SharedElement && value)
         {
             std::unique_lock lock(mut_);
-            auto [_, created] = map_.insert_or_assign(key, std::forward<SharedElement>(value));
+            auto [_, created] = map_.insert_or_assign(key, std::move(value));
```

Comment:
it is fine to use `forward `

Reply:
std::forward usually used in template. Our use of std::forward is not standardized.